client/my-sites/patterns/components/type-toggle/index.tsx

```diff
@@ -0,0 +1,49 @@
+import { useTranslate } from 'i18n-calypso';
+import {
+	PatternLibraryToggle,
+	PatternLibraryToggleOption,
+} from 'calypso/my-sites/patterns/components/toggle';
+import { usePatternsContext } from 'calypso/my-sites/patterns/context';
+import { getCategoryUrlPath } from 'calypso/my-sites/patterns/paths';
+import { PatternTypeFilter } from 'calypso/my-sites/patterns/types';
+
+type TypeToggleProps = {
+	onChange?( value: PatternTypeFilter ): void;
+};
+
+export function TypeToggle( { onChange }: TypeToggleProps ) {
+	const translate = useTranslate();
+	const { category, isGridView, patternTypeFilter } = usePatternsContext();
+
+	return (
+		<PatternLibraryToggle onChange={ onChange } selected={ patternTypeFilter }>
+			<PatternLibraryToggleOption
+				className="pattern-library__toggle-option--type"
```

Comment:
I understand that it's only 1 className here, but I would move styles for it inside `/components/type-toggle/styles.scss`, to have everything in one place.

The same for `view-toggle`

Reply:
That's fair – I'll stick to the principle 👍